automation/terraform/testnets/watchdog-test/community_fast_locked_keys.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
Make sense to check these in?

Reply:
What should I do usually for testnets? For some I think we've said check in / have had to check in (like testworld) , but in general should I not be?

Reply:
It's helpful to check all of the files in for debuggability and transparency, so I don't mind this as long as you intend to keep using this testnet as you develop the watchdog. But once this testnet outlives its utility and gets shutdown, it should be removed from the repo as well.

Reply:
Sounds good, makes sense to me (and yep that's the idea)